cypress/integration/premium/teamflow.spec.ts

```diff
@@ -43,28 +43,30 @@ describe("Teams flow (seeded)", () => {
       cy.loginWithCySession();
       cy.visit("/settings/teams");
     });
-    it("edits an existing team", () => {
+    it("edit name of an existing team", () => {
```

Comment:
```suggestion
    it("edits name of an existing team", () => {
```
I don't actually know what the convention is for the `it` text so feel free to disregard if you prefer the infinitive over the present tense. I personally just liked how it read more like a sentence.